doc/exception_interrupts.rst

```diff
@@ -3,28 +3,17 @@
 Exceptions and Interrupts
 =========================
 
-Ibex currently implements a free-form vectored trap handler mechanism for interrupts and exceptions.
-The base address of the interrupt vector table is given by the boot address (must be aligned to 256 bytes, i.e., its least significant byte must be 0x00).
+Ibex currently implements trap handling for interrupts and exceptions according to the RISC-V Privileged Specification, draft version 1.11.
```

Comment:
It's actually ratified now

Reply:
Thanks, fixed that and also in other places in the doc.